specification/containerservices/resource-manager/Microsoft.ContainerService/preview/2018-09-30-preview/openShiftManagedClusters.json

```diff
@@ -0,0 +1,590 @@
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/openShiftManagedClusters/{resourceName}": {
+      "get": {
+        "tags": [
+          "OpenShiftManagedClusters"
+        ],
+        "operationId": "OpenShiftManagedClusters_Get",
+        "summary": "Gets a openshift managed cluster.",
+        "description": "Gets the details of the managed openshift cluster with a specified resource group and name.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the openshift managed cluster resource."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/OpenShiftManagedCluster"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get OpenShift Managed Cluster": {
+            "$ref": "./examples/OpenShiftManagedClustersGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "OpenShiftManagedClusters"
+        ],
+        "operationId": "OpenShiftManagedClusters_CreateOrUpdate",
+        "summary": "Creates or updates an openshift managed cluster.",
+        "description": "Creates or updates a openshift managed cluster with the specified configuration for agents and OpenShift version.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "resourceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the openshift managed cluster resource."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/OpenShiftManagedCluster"
+            },
+            "description": "Parameters supplied to the Create or Update an OpenShift Managed Cluster operation."
+          }
+        ],
+        "responses": {
```

Comment:
please model the default response to capture all the error states. Sample can be found in batch RP swagger.

Reply:
Schema has not been added for the default response. Please check Batch RP swagger for sample.